requirements.txt

```diff
@@ -6,6 +6,7 @@ etils[epath_no_tf]
 cloudml-hypertune
 google-cloud-storage
 imageio
+imageio-ffmpeg
```

Comment:
I don't think this should be a mandatory kubric dependency just because the dynamic nerf worker uses it. I'd prefer adding specific instructions for additional dependencies to that worker.